### DISCUSSION TITLE
Why are there no links to a profiles Gists page?

### DISCUSSION THREAD

**Discussion by rBrenick** [community/community#59853] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's not a ton of work to manually type in the `gist.` URL prefix, but a link would surely be more convenient. 

I do see that there was [a discussion about this 4 years ago.](https://github.com/orgs/community/discussions/22691). But unfortunately didn't seem to result in anything.

This has been bothering me for years, so I figured I might as well ask, why?

**Comment by 3urobeat** [comment]
Well, good question, it is indeed something that should be added imo  
If you want to show Gists on your GitHub profile you can create one that links to your other Gists and pin it.  
I did exactly that and while it takes a little bit of extra effort that shouldn't be necessary it works quite well
